GetStream/stream-chat-android
# [4044] Fix avatar in navigation drawer

### 🎯 Goal

Closes #4044 

We need to observe the currently logged in user in he drawer to ensure that the user avatar is always displayed correctly.

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/184893796-10ed7260-5c5e-4911-87a9-439029807844.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/184893820-d98564cd-244f-4b16-88e5-36323865ebca.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Login with any user
2. Check the user avatar in navigation drawer
3. Logout and login with a different user
4. Check the user avatar in navigation drawer

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
